src/message-stream.ts

```diff
@@ -115,11 +123,15 @@ export class ChannelError extends Error implements ServiceError {
  *     {@link https://nodejs.org/en/docs/guides/backpressuring-in-streams/} for
  *     more details.
  * @property {number} [maxStreams=5] Number of streaming connections to make.
+ * @property {number} [pullDeadline=900000] Deadline to be applied to each
```

Comment:
The internal "deadline" that we use through gapic wants a time valur, but we repurpose "deadline" here to mean the amount of time from now. Should we just have the user provide a time value as well, or maybe rename this setting to "streamAliveMs" (or something)?

Reply:
We actually bypass the gapic for this class and use plain old grpc. Definitely not partial to the name though

Reply:
Ah, ok. But I think the naming logic is still the same, where deadline is a time stamp in gRPC land, whereas that's not how the new option would be used. I think the name should not include "deadline" only for that reason. As far as other names, I'm open to ideas as well!

Reply:
sure, `pullTimeout` then?

Reply:
That makes sense to me!

Reply:
Done. PTAL!

Reply:
Thank you sir!